backend/infrahub/workflows/constants.py

```diff
@@ -1,6 +1,21 @@
+from typing import Any
+
 from infrahub.utils import InfrahubStringEnum
 
 
 class WorkflowType(InfrahubStringEnum):
     INTERNAL = "internal"
     USER = "user"
+
+
+TAG_NAMESPACE = "infrahub.app"
+
+
+class WorkflowTag(InfrahubStringEnum):
+    BRANCH = "branch/{branch_name}"
+    WORKFLOWTYPE = "workflow-type/{workflow_type}"
+    DATABASE_CHANGE = "database-change"
+
+    def render(self, **kwargs: Any) -> str:
+        rendered_value = str(self.value).format(**kwargs)
+        return f"{TAG_NAMESPACE}/{rendered_value}"
```

Comment:
This looks a bit weird to me that you'd have to know what parameter to use for the render method and that it would be different depending on the enum value. 

Something like this would be more clear:

```python
class WorkflowTag(InfrahubStringEnum):
    BRANCH = "branch/{identifier}"
    WORKFLOWTYPE = "workflow-type/{identifier}"
    DATABASE_CHANGE = "database-change"

    def render(self, identifier: str | None = None) -> str:
        if identifier is None:
            return f"{TAG_NAMESPACE}/{self.value}"
        rendered_value = str(self.value).format(identifier=identifier)
        return f"{TAG_NAMESPACE}/{rendered_value}"
```

But I'm not sure this should even be an enum. Later it's called with just `tags.append(tag.render())` which would work for WorkflowTag.DATABASE_CHANGE but not the other ones.

Reply:
Good point, I made the change